urlshort/handler.go

```diff
@@ -12,7 +14,14 @@ import (
 // http.Handler will be called instead.
 func MapHandler(pathsToUrls map[string]string, fallback http.Handler) http.HandlerFunc {
 	//	TODO: Implement this...
-	return nil
+	return func(w http.ResponseWriter, r *http.Request) {
+		path, ok := pathsToUrls[r.URL.Path]
+		if ok {
+			http.Redirect(w, r, path, http.StatusFound)
```

Comment:
Is `http.StatusFound` correct? I remember needing to use a `211` or some other status to signify a redirect.

Reply:
For the status code, I referred the following article https://stackoverflow.com/questions/22887148/is-the-http-statusfound-the-correct-status-for-a-click-redirect

Reply:
Checks out